src/utils.rs

```diff
@@ -132,6 +136,38 @@ pub(crate) fn write_wallet_from_mnemonic_and_password(
     .map(|_| ())
 }
 
+/// Determine whether to replace an existing wallet
+///
+/// If return true and there is an existing wallet, will be remove first
+/// 
+/// If return false, nothing changes
+pub(crate) fn should_replace_wallet(wallet_path: &Path, force: bool, mut reader: impl BufRead) -> bool {
+    if wallet_path.exists() {
+        if force {
+            fs::remove_file(wallet_path).unwrap();
```

Comment:
Perhaps we should also warn the user that this is happening.

```suggestion
        if force {
            println_warning("Because the `--force` argument was supplied, the wallet at {} will be removed.", wallet_path.display());
            fs::remove_file(wallet_path).unwrap();
```